### ISSUE TITLE
Port request for Openlayers

### ISSUE THREAD

**Issue by djzwerg** [backdrop-ops/contrib#458]
**Name of the module, theme, or layout**
Openlayers

**Link to the drupal.org module, theme, or layout**
https://www.drupal.org/project/openlayers

**Comment by olafgrabienski** [comment]
I guess that's quite a big module and not easy to port. However, I have the impression that people from the Backdrop community are or were involved in Openlayers for Drupal. Maybe ask in our [Zulip live chat](https://backdrop.zulipchat.com/) if someone is able to help with the port or interested to do it for a bounty?

**Comment by djzwerg** [comment]
@olafgrabienski you're right. I have not enough coding skills to port that my way.  I'm going to try the chat but I don't know it yet.
